src/components/upload_stepper/UploadAgreement.js

```diff
@@ -0,0 +1,79 @@
+/** @jsx jsx */
+import {jsx} from "@emotion/core";
+import {useSelector} from "react-redux";
+import Button from "@material-ui/core/Button/Button";
+import withStyles from "@material-ui/core/styles/withStyles";
+import colors from "../../styles/colors";
+import textStyles from "../../styles/text";
+import List from "@material-ui/core/List";
+import IconListItem from "../widgets/list_items/IconListItem";
+import EditIcon from "@material-ui/icons/Edit";
+import WarningIcon from "@material-ui/icons/Warning";
+
+const CustomUploadButton = withStyles({
+  root: {
+    backgroundColor: colors.presqtBlue,
+    '&:hover': {
+      backgroundColor: '#0a4996',
+    },
+  },
+})(Button);
+
+/**
+ * Component responsible for rendering the upload button in the upload stepper and passing the
+ * selected file to the Upload API endpoint
+ **/
+export default function UploadAgreement({handleNext, resourceToUploadTo}) {
```

Comment:
Need resourceToUploadTo param?

Reply:
Yep, to determine the text to display in the actions list